### ISSUE TITLE
Package uses different electron version than electron-prebuilt

### ISSUE THREAD

**Issue by stefanKuijers** [chentsulin/electron-react-boilerplate#527]
## Downgrade
I was forced to downgrade electon to 1.2.8 because I want to make integrate node-sass into my app. 

So I changed package.json to include electron-prebuild 1.2.8. Deleted the electron-prebuilt folder and ran npm install. 
I changed ```node_modules/.bin/electron``` to point to ```electron-prebuilt/cli.js``` instead of ```electron/cli.js```.

When I run ```npm run dev``` and check the ```process.versions```.
```
Node: 6.1.0
Electron: 1.2.8
modules (V8): 48
```

## Problem
The problem comes after I pack the app. ```npm run package-win```. **The packing runs as expected** _however_ I do see that in the logs it used electron version 1.4.5. I checked the process.versions in the running packaged app and it returns the following:
```
Node: 6.5.0,
Electron: 1.4.5
modules (V8): 50
```

In any other case fine except that node-sass won't work which is one of the main features of the app. 

## Solution
I found ```node_modules/electron``` and deleted it. After that I repackaged my app and it build with the right version of electron.

Because of these issues I assumed that it would automatically pickup the same version as I installed with electron-prebuilt:
https://github.com/webpack/webpack/issues/1463
https://github.com/chentsulin/electron-react-boilerplate/commit/ef2e24fee1e8ac4285e1068d418e688d1c0a16bb
https://github.com/chentsulin/electron-react-boilerplate/issues/33

Just opening an issue cause it might be good to add this to the docs or in case someone else is forced to downgrade.

**Comment by amilajack** [comment]
This seems like more of a question for https://github.com/electron-userland/electron-builder

**Comment by stefanKuijers** [comment]
Thank you. I will be looking into this. My main aim in opening an issue is to make some steps more clear for others cause I'm running into quite a few (small) questions. For now for example I would like to keep the electron-debug (chrome inspector) open for a packaged app. 

I know that should be a different issue but until now I did not figure out how to do that. Do you guys have some kind of FAQ that we can contribute to?

**Comment by amilajack** [comment]
Let me know if you have resolved this. It may be a question for future users of the boilerplate

**Comment by stefanKuijers** [comment]
For now the solution that I proposed before is enough for me.
> I found node_modules/electron and deleted it. After that I repackaged my app and it build with the right version of electron.

We should keep in mind that this a an edge-case. 80%+ of the users will want to use the latest version of Electron. 10% might downgrade but not as many versions back as I had to. This was for a very specific purpose: working with node-sass. based on that:
I'm not sure I would put time into finding a better solution as deleting one folder and maybe changing 4 paths over 2 files. 

Makes sense right?
